### DISCUSSION TITLE
Installation on M2 ventura 13

### DISCUSSION THREAD

**Discussion by mike235711** [Homebrew/discussions#4628] [Getting started]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.28
ORIGIN: https://github.com/Homebrew/brew
HEAD: 2b4cfb033757b63ab0b128d01c58db76480956f9
Last commit: 24 hours ago
Core tap JSON: 07 Jul 13:36 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 13.3-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Settings.

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 14.3.


Warning: Your Xcode (14.2) is outdated.
Please update to Xcode 14.3 (or delete it).
Xcode can be updated from the App Store.
```

### Description of issue

Homebrew by default, on the website it says that when installing for a mac, it will install itself inside the opt folder. However, after following the steps of installation, and apparently getting a succesful installation, the folder doesnt seem to appear in the specified location (users/MyUser/opt). Hence when downloading packages with homebrew I can download them, but I cant locate them.

I imagine that it might install the homebrew folder in another directory, but when searching for it on Finder it doesn't appear.

These are the commands I used when installing:
% export HOMEBREW_PREFIX=/Users/miguel/opt
% /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

Note: I also tried to install it (after uninstalling) without changing the homebrew_prefix and I get the same result. 

Thanks

**Comment by carlocab** [comment]
> When I install homebrew, it installs succesfully but then the file doesnt appear in opt.

What is the file you're referring to here when you say "the file doesnt appear"?

**Reply by mike235711** [reply]
homebrew. I guess it should be a folder, not a file, sorry.

**Reply by carlocab** [reply]
What commands did you run when you expected homebrew to appear but it did not?

**Reply by mike235711** [reply]
I could run commands on the terminal to access opt, but I simply manually accessed the opt folder (through finder) and it didn't show up. All this right after making the installation.

**Reply by carlocab** [reply]
That's too vague for me to really be able to tell what you're doing exactly, so I'm afraid I can't help you.

If you'd like for me to try a bit harder, please provide precise and detailed instructions describing what you did, what happened, and what you expected to happen.

Otherwise, you can probably take comfort in the knowledge that Homebrew is installed and useable, because you were able to call `brew config` and `brew doctor`.

**Reply by mike235711** [reply]
Ill try to explain the problem in more detail. Homebrew by default, on the website it says that when installing for a mac, it will install itself inside the opt folder. However, after following the steps of installation, and apparently getting a succesful installation, the folder doesnt seem to appear in the specified location (users/MyUser/opt). Hence when downloading packages with homebrew I can download them, but I cant locate them. 

I imagine that it might install the homebrew folder in another directory, but when searching for it on Finder it doesn't appear. 

These are the commands I used when installing:
% export HOMEBREW_PREFIX=/Users/miguel/opt       
 % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

Note: I also tried to install it (after uninstalling) without changing the homebrew_prefix and I get the same result.

**Reply by carlocab** [reply]
On M1 macOS, the install script always installs into `/opt/homebrew`. You should look for it there, instead of at `/Users/YourUserName/opt/homebrew`.

There really is no need to look at your Homebrew installation using Finder, but, if you must, you can access it by using "Go to Folder..." (you can find this under the "Go" menu on the menu bar, or by pressing `Shift + Command + G`) and then using to to go to `/opt/homebrew`.

You can also type
```
open /opt
```
in a terminal window to open the `/opt` directory in Finder.

**Reply by mike235711** [reply]
Yes, that is the same opt folder I have been saying it has no hombrew folder or file.  That's how I have been accessing it all along, but its empty. 
Maybe this clarifies that the opt folder has no hombrew:
% cd opt
% ls
output: anaconda3

**Reply by carlocab** [reply]
It's not the same. You need to look in `/opt`, not `opt`. Try:
```
ls -la /opt
```

**Reply by mike235711** [reply]
Thanks, got it!! I think the problem was that I was writting \ instead of /. Appreciate the fast replies :).
